src/utils/log.js

```diff
@@ -14,6 +14,20 @@
  * limitations under the License.
  */
 
+ /**
+  * Debug logger, only log message if #swg.log=1
+  * @param {...*} var_args [decription]
+  */
+export function debugLog(var_args) {
+  if (/swg.dbg=1/.test(self.location.hash)) {
```

Comment:
I'd rather call it `swg.debug`. Also, do you think we should calculate it once and cache?

Reply:
will change the dbg -> debug.

Having it be a cache value makes testing harder and since it's not in any loops I left it as the simple test..